src/main/java/me/profelements/dynatech/items/tools/Orechid.java

```diff
@@ -46,8 +46,13 @@ public void tick(Block b) {
                 
                 Block relBlock = b.getRelative(relative);
     
-                if (oreMap.containsKey(relBlock.getType())) {
-                    ItemStack item = oreMap.get(relBlock.getType()).getRandom();
+                if (ORE_MAP.containsKey(relBlock.getType())) {
+                    RandomizedSet<ItemStack> set = ORE_MAP.get(relBlock.getType());
+                    if (set == null) {
+                        continue;
+                    }
+
+                    ItemStack item = ORE_MAP.get(relBlock.getType()).getRandom();

```

Comment:
Here you are performing the same `get` operation THREE times.
Firstly, `.containsKey()` is just a null check on a get operation, then you do a get operation and do a null check yourself, only to do yet another get operation. This can be simplified and optimized a lot here.

```suggestion
                RandomizedSet<ItemStack> set = ORE_MAP.get(relBlock.getType());
                if (set != null) {
                    ItemStack item = set.getRandom();

```